src/test/java/com/onaio/steps/handler/ExceptionHandlerTest.java

```diff
@@ -57,13 +65,17 @@ public void testHandleMultipleExceptionScenarios() {
         verify(handler, times(1)).postDelayed(acRunnable.capture(), eq(1000L));
 
         acRunnable.getValue().run();
-        verify(menuHandler, times(1)).open();
+        verify(resolvableException, times(1)).tryToResolve();
 
         exceptionHandler.handle(mock(NullPointerException.class));
         verifyAlertError(R.string.something_went_wrong_try_again);
+
+        verify(callback, times(3)).onDismiss(any(Exception.class), anyInt());
```

Comment:
Nice work on this :clap: 

Change: Verify the specific exceptions here and not just any exception

Reply:
This was not done. Make sure this is completed  or an issue is created

Reply:
i've verified specific exceptions could your bit more elaborate on what kind of change is expected.

Reply:
`verify(callback, times(3)).onDismiss(any(NullPointerException.class), anyInt());`

Reply:
fixed here #170